src/log-query/src/log_query.rs

```diff
@@ -34,6 +36,18 @@ pub struct LogQuery {
     pub context: Context,
 }
 
+impl Default for LogQuery {
+    fn default() -> Self {
+        Self {
+            table: TableName::new("", "", ""),
+            time_filter: Default::default(),
+            columns: vec![],
+            limit: None,
+            context: Default::default(),
+        }
+    }
+}
+
```

Comment:
How about removing the `impl Default`? It's strange to have a table name with empty strings. We can create a `new` function if needed.

Reply:
This is for deserializing, when some fields are missing in the incoming query payload